profiles/Vervaine-setup.yaml

```diff
@@ -0,0 +1,196 @@
+---
+# https://elanthipedia.play.net/Moon_mage#Circle_Requirements
+# 4 Magics: Augmentation, Attunement, Arcana, Targeted Magic/Debilitation (Does not include Lunar Magic or Astrology)
+# 4 Survivals: Perception, Athletics, Evasion, Skinning (Does not include Thievery)
+# 3 Lores: Appraisal, Tactics, Mechanical Lore (Does not include Scholarship)
+# 2 Additional: Astrology, Scholarship
+
+# It is recommended to get a target or debilitation spell for your second circle. (e.g. Dinazen Olkar or Calm)
+# Please see the 'offensive_spells:' setting below for a commented (unused) example. Uncomment to begin usage (remove #'s).
+
+# Eventually you will need to train 3 more Magics to circle (for a total of 7)
+# Eventually you will need to train 1 more Survival to circle (for a total of 5)
+
+# Hunting settings
+hometown: Crossing
+training_manager_hunting_priority: true
+training_manager_priority_skills:
+- Brawling
+- Small Edged
+- Shield Usage
+hunting_info:
+- :zone: wood_trolls
+  stop_on:
+  - Brawling
+  - Shield Usage
+  - Evasion
+  - Skinning
+  - Tactics
+fatigue_regen_threshold: 85
+fatigue_regen_action: bob
+weapon_training:
+  Small Edged: rapier
+  Brawling: ''
+
+# Gear settings
+gear:
+# - :adjective: example
+#   :name: gear
+#   :is_leather: true
+#   :hinders_lockpicking: false
+#   :is_worn: true
+- :adjective: firesilk
+  :name: robe
+  :is_leather: true
+  :hinders_lockpicking: false
+  :is_worn: true
+- :adjective: chain
+  :name: balaclava
+  :is_leather: true
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: target
+  :name: shield
+  :is_leather: false
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: parry
+  :name: stick
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: true
+- :adjective: steel
+  :name: knuckles
+  :is_leather: false
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: elbow
+  :name: spikes
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: true
+- :adjective: steel
+  :name: rapier
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: false
+gear_sets:
+  standard:
+  - chain balaclava
+  - firesilk robe
+  - chain gloves
+  - target shield
+  - parry stick
+  - steel knuckles
+  - elbow spikes
+picking_box_source: bag
+braid_item: grass
+listen: true
+gem_pouch_adjective: shadowleaf
+
+# Combat settings
+
+# Uncomment (remove #'s) to train Targeted Magic/Debilitation with the listed spells (or change to use different spells).
+offensive_spells:
+- skill: Targeted Magic
+   name: Dinazen Olkar
+   abbrev: do
+   cast_only_to_train: true
+   mana: 2
+   - skill: Debilitation
```

Comment:
You have an indentation issue here, it should line up with the -skill above it

Reply:
fixed :) Thanks!